src/main/java/ru/practicum/shareit/booking/service/impl/BookingServiceImpl.java

```diff
@@ -114,23 +119,31 @@ public BookingResponseDTO findBookingById(Long bookingId, Long userId) {
                     String.format("Пользователь с Id: %d не является владельцем предмета c Id: %d или брони c Id: %d",
                             userId, booking.getItem().getId(), booking.getId()));
         }
-        return BookingMapper.toBookingResponseDTO(booking);
+        return toBookingResponseDTO(booking);
     }
 
     @Override
     @Transactional
```

Comment:
Я не могу быть на 100% уверен в том что сейчас напишу ). Но как я понял что данная анатация очень даже необходима если в одном запросе происходит обращение к базе больше одного раза.  Поправте меня если я не прав и стоит в коде поправить это.

Reply:
Чисто теоретически Да, такое может быть, но только для высоко нагруженных систем и очень редко. На практике я ни разу не сталкивался. 
.------------------------------

In a highly concurrent application it could (theoretically) happen that data you've read in the first select is modified before the other selects are executed.If that is a situation that could occur in your application you should use a transaction to wrap your selects. Make sure you pick the correct isolation level though, not all transaction types guarantee consistent reads.